web/docs/guides/auth/auth-notion.mdx

```diff
@@ -14,51 +14,55 @@ To enable Notion Auth for your project, you need to set up a Notion Application
 Setting up Notion logins for your application consists of 3 parts:
 
 - Create and configure a Notion Application [Notion Developer Portal](https://www.notion.so/my-integrations)
-- Add your Notion OAuth Consumer keys to your [Supabase Project](https://app.supabase.com)
+- Retrieve your OAuth client ID and OAuth client secret and add them to your [Supabase Project](https://app.supabase.com)
 - Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
 
 ## Steps
 
-### Access your Notion account
+### Create your notion integration
 
-- Go to [notion.so](https://notion.so/).
-- Click on `Login` at the top right to log in.
+- Go to [developers.notion.com](https://developers.notion.com/).
+- Click "View my integrations" and login.
 
 ![notion.so](/img/guides/auth-notion/notion.png)
 
-- Once logged in, go to [notion.so/my-integrations](https://notion.so/my-integrations).
+- Once logged in, go to [notion.so/my-integrations](https://notion.so/my-integrations) and create a new integration.
+- When creating your integration, ensure that you select "Public integration" under "Integration type" and "Read user information including email addresses" under "Capabilities".
+- You will need to add a redirect uri, see [Add the redirect uri](### Add the redirect uri)
```

Comment:
Should there be a link/uri associated with [Add the redirect uri] ?

Reply:
hmm i'm trying to link to a section in the page actually and it seems this is the way to do it in mdx?

Reply:
Ah okay, wasn't aware of this.  Let's go with this then!